lib/src/model/game/game_controller.dart

```diff
@@ -113,6 +114,16 @@ class GameController extends _$GameController {
 
         _socketEventVersion = fullEvent.socketEventVersion;
 
+        // Play "dong" sound when this is a new game and we're playing it (not spectating)
+        final myUserId = ref.watch(authSessionProvider)?.user.id;
```

Comment:
You don't need to watch session, you already have `game.youAre` which tell the side you're playing.